Header header
string robot_id
bool has_arrived